src/tasks/MobileBuildTasks/Apple/AppleProject.cs

```diff
@@ -0,0 +1,162 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Mobile.Build.Clang;
+
+namespace Microsoft.Apple.Build
+{
+    public sealed class AppleProject
+    {
+        private const string DefaultMinOSVersion = "11.0";
```

Comment:
please update the comment in the root Directory.Build.props (look for SetOSTargetMinVersions) to also bump this when bumping the version.